web_responsive_pwa/static/src/xml/pwa_install.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates>
+    <t t-name="web_responsive_pwa.systray.install">
+        <li class="o_mail_systray_item" id="pwa_install_item" hidden="1">
+            <a title="Install PWA" href="#" role="button" id="pwa_install_button">
+                <i class="fa fa-download" role="img" aria-label="Install PWA"/>
```

Comment:
Don't mobile browsers already include a button like this natively?

Reply:
Yes they do, this `Install PWA` button is at the moment little invasive, see the download button in the navigation bar and the "Add tho home screen" banner at the bottom:

![Screenshot_20200519-153008](https://user-images.githubusercontent.com/1033131/82337278-e36a9a00-99eb-11ea-92de-947654cc36e2.jpg)


I added the button to give the opportunity to extend it with custom behaviors (like a more evident modal dialog for example).
I would like to read the opinion of other contributors about the default behavior of this functionality. We could also think to make it configurable (more or less invasive)...

Reply:
I think that if browsers supply a native UI for this, we should remove it.

Reply:
I am not sure how to check this, unless we totally remove the `beforeinstallprompt` event handler.
I don't know if there are cases or browsers where the native UI does not show while `beforeinstallprompt` is fired (showing our "download" icon only).
Since the icon in this implementation is little invasive, I think we could keep it.

Reply:
Hi @eLBati. Regarding this button : what about to move it and set instead an new option in the top right dropdown menu 

My PoV : 
- this button is not usefull a lot of time, so it's not interesting to have it as a shortcut, allways accessible
- the icon is not very intuitive : it means "download" and the action is more an "installation"
- the alt text "Install PWA" is not available on mobile, so there's no way of knowing what this button is for, before clicking on it.

something like that : with "Install PWA" or "Install App"

![image](https://user-images.githubusercontent.com/3407482/83075308-ed3b6f80-a073-11ea-9b91-c4d6529eef0c.png)

What do you think ? Just a suggestion, not a blocking point.

kind regards.

Reply:
@legalsylvain I think you are right. Done:

![Screenshot from 2020-05-28 12-22-05](https://user-images.githubusercontent.com/1033131/83129974-e1849300-a0dd-11ea-84a1-5e1fb2d72c89.png)